composer.json

```diff
@@ -32,8 +32,8 @@
         "chameleon-system/code-style-config": "dev-master@dev",
         "chameleon-system/sanitycheck": "~7.0.0",
         "chameleon-system/sanitycheck-bundle": "~7.0.0",
-        "phpunit/dbunit": "~2.0",
-        "phpunit/phpunit": "~5.7"
+        "phpunit/phpunit": "^7.5.20",
```

Comment:
To avoid dependency conflicts, I had to update to phpunit 7.x. All tests still run.